x-pack/plugins/cases/public/components/all_cases/all_cases_list.test.tsx

```diff
@@ -131,69 +131,56 @@ describe.skip('AllCasesListGeneric', () => {
     window.localStorage.clear();
   });
 
+  afterEach(() => {
+    moment.tz.setDefault('Browser');
+  });
+
   it('should render AllCasesList', async () => {
     useLicenseMock.mockReturnValue({ isAtLeastPlatinum: () => true });
 
-    const wrapper = mount(
-      <TestProviders>
-        <AllCasesList />
-      </TestProviders>
-    );
+    appMockRenderer.render(<AllCasesList />);
 
     await waitFor(() => {
-      expect(wrapper.find(`a[data-test-subj="case-details-link"]`).first().prop('href')).toEqual(
-        `/app/security/cases/test`
+      expect(screen.getAllByTestId('case-details-link')[0]).toHaveAttribute(
+        'href',
+        '/app/security/cases/test'
       );
-      expect(wrapper.find(`a[data-test-subj="case-details-link"]`).first().text()).toEqual(
+      expect(screen.getAllByTestId('case-details-link')[0]).toHaveTextContent(
         useGetCasesMockState.data.cases[0].title
       );
       expect(
-        wrapper.find(`span[data-test-subj="case-table-column-tags-coke"]`).first().prop('title')
-      ).toEqual(useGetCasesMockState.data.cases[0].tags[0]);
-      expect(
-        wrapper.find(`[data-test-subj="case-user-profile-avatar-damaged_raccoon"]`).first().text()
-      ).toEqual('DR');
-      expect(
-        wrapper
-          .find(`[data-test-subj="case-table-column-createdAt"]`)
-          .first()
-          .childAt(0)
-          .prop('value')
-      ).toBe(useGetCasesMockState.data.cases[0].createdAt);
-
-      expect(wrapper.find(`[data-test-subj="case-table-case-count"]`).first().text()).toEqual(
-        'Showing 10 cases'
+        screen.getAllByTestId('case-user-profile-avatar-damaged_raccoon')[0]
+      ).toHaveTextContent('DR');
+      expect(screen.getAllByTestId('case-table-column-tags-coke')[0]).toHaveAttribute(
+        'title',
+        useGetCasesMockState.data.cases[0].tags[0]
       );
+      expect(
+        screen.getAllByTestId('case-table-column-createdAt')[0].querySelector('.euiToolTipAnchor')
+      ).toHaveTextContent(removeMsFromDate(useGetCasesMockState.data.cases[0].createdAt));
```

Comment:
we are removing milliseconds in `CreatedAt` column in `use_cases_columns` hook, while mock data is with milliseconds.